specification/azsadmin/resource-manager/storage/Microsoft.Storage.Admin/preview/2016-05-01/queueServices.json

```diff
@@ -0,0 +1,198 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "2015-12-01-preview",
+        "title": "StorageManagementClient",
+        "description": "The Admin Storage Management Client."
+    },
+    "host": "adminmanagement.local.azurestack.external",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Storage.Admin/farms/{farmId}/queueservices/default": {

```

Comment:
<I left similar comment for tableServices, copy pasting here for easy navigation>

I understand that `default` is the only valid value now but right thing to do here is model it as a parameter with single-value enum instead of hardcoding.


```
/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers
/Microsoft.Storage.Admin/farms/{farmId}/queueservices/{queueServiceType}
```


and model {queueServiceType} as:

```
{
    "name": "queueServiceType",
    "in": "path",
    "description": "The queue service type. Should always be \"default\".",
    "required": true,
    "type": "string",
    "enum": [
       "$default"
    ],
    "x-ms-enum": {
        "name": "QueueServiceType",
        "modelAsString": true
    }
}
```

Since this is an enum with `ONE value` (default) and is `required` there will not be an enum type generated in SDKs instead method impl will uses a const with value `default`, hence it is not exposed to user i.e.  generated methods will not have a "queueServiceType" parameter, which is exactly what you want here.

This is the guideline on how paths should be structured. So often services actually have a "parameter" in that spot (rather than a word like "default"..), only that there is only one valid value so far. By following above guideline, swagger mirrors the service's semantics a little more closely than if the parameter was hardcoded into the path.

Note: I'm not sure `queueServiceType` is the right param name, or just `name` will do here, since you know service better you should be able to come-up with right name for the parameter.